.editorconfig

```diff
@@ -242,6 +242,26 @@ dotnet_diagnostic.IDE0130.severity = silent
 # IDE0150: Null check can be clarified
 dotnet_diagnostic.IDE0150.severity = silent
 
+# IDE2000: Allow multiple blank lines
+dotnet_style_allow_multiple_blank_lines_experimental = false
+dotnet_diagnostic.IDE2000.severity = warning
```

Comment:
Don’t these rules have category Style? If so, line 185 should already escalate these to warning severity?

Reply:
Yeah indeed they are not needed. I even had some of these entries as "silent" by mistake. There are more violations to fix.